### DISCUSSION TITLE
Why it shows me someone's else github stats?

### DISCUSSION THREAD

**Discussion by CondratKenasil** [anuraghazra/github-readme-stats#2292] [Q&A] [answered]
![Screenshot 2022-11-24 124937](https://user-images.githubusercontent.com/118928487/203766032-7186c434-6501-4e09-a17d-e3e24a5f7347.png)

**Comment by MitkoVtori** [comment, marked as answer]
You need to change the name:
![Screenshot 2022-11-24 122512](https://user-images.githubusercontent.com/112943652/203766862-2b8a11f1-cfe7-43e1-a176-d80df9dd11d1.png)
As you can se, in blue the `?username=anuraghazra`, change it to `?username=CondratKenasil`
Here:
``![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=CondratKenasil&show_icons=true&theme=transparent)``
![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=CondratKenasil&show_icons=true&theme=transparent)
